C:\Users\benjm\git\codeHistoriesPythonProjects\Pokedex> & C:/Users/benjm/AppData/Local/Programs/Python/Python310/python.exe c:/Us
ers/benjm/git/codeHistoriesPythonProjects/Pokedex/pokedex.pyX
X
DevTools listening on ws://127.0.0.1:51134/devtools/browser/c9ce9561-7ac8-44c8-a2bd-fa9bc9a8a2a3X
X> c:\users\benjm\git\codehistoriespythonprojects\pokedex\pokedex.py(96)update_gui_fields()
-> img = ImageTk.PhotoImage(Image.open(poke_data['image_url']))X
(Pdb)Xim  poke) _data["image_url"]
X'https://img.pokemondb.net/sprites/sword-shield/icon/venusaur.png'
(Pdb)Xcontinu       continue
XException in Tkinter callback
Traceback (most recent call last):X
  File "C:\Users\benjm\AppData\Local\Programs\Python\Python310\lib\tkinter\__init__.py", line 1921, in __call__X
    return self.func(*args)X
  File "c:\Users\benjm\git\codeHistoriesPythonProjects\Pokedex\pokedex.py", line 83, in get_pokemonX
    update_gui_fields()X
  File "c:\Users\benjm\git\codeHistoriesPythonProjects\Pokedex\pokedex.py", line 96, in update_gui_fieldsX
    img = ImageTk.PhotoImage(Image.open(poke_data['image_url']))X
  File "C:\Users\benjm\AppData\Local\Programs\Python\Python310\lib\site-packages\PIL\Image.py", line 3068, in openX
    fp = builtins.open(filename, "rb")X
OSError: [Errno 22] Invalid argument: 'https://img.pokemondb.net/sprites/sword-shield/icon/venusaur.png'X
X(Pdb) continueX
Exception in Tkinter callbackX
Traceback (most recent call last):X
  File "C:\Users\benjm\AppData\Local\Programs\Python\Python310\lib\tkinter\__init__.py", line 1921, in __call__X
    return self.func(*args)X
  File "c:\Users\benjm\git\codeHistoriesPythonProjects\Pokedex\pokedex.py", line 83, in get_pokemonX
    update_gui_fields()X
  File "c:\Users\benjm\git\codeHistoriesPythonProjects\Pokedex\pokedex.py", line 96, in update_gui_fieldsX
    img = ImageTk.PhotoImage(Image.open(poke_data['image_url']))X
  File "C:\Users\benjm\AppData\Local\Programs\Python\Python310\lib\site-packages\PIL\Image.py", line 3068, in openX
    fp = builtins.open(filename, "rb")X
OSError: [Errno 22] Invalid argument: 'https://img.pokemondb.net/sprites/sword-shield/icon/venusaur.png'X
XPS 